certman.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.required_ruby_version = '>= 2.2'
+  spec.required_ruby_version = '>= 2.3'
```

Comment:
I want to use `&.` , too. but I want to support Ruby 2.2 ...

Please give me some time.

Reply:
Fixed in the last push! I didn't realize 2.2 was a hard requirement.